packages/mobile/src/transactions/feed/TransferFeedItem.tsx

```diff
@@ -35,14 +37,14 @@ function TransferFeedItem({ transfer }: Props) {
   }
 
   let { title, subtitle, recipient } = useTransferFeedDetails(transfer)
-  const isPending = transfer.status === TransactionStatus.Pending
+  const isStandbyTransaction = transfer.status === TransactionStatus.Pending
 
   // I feel this should be inside useTransferFeedDetails() instead of here. What do you think?
-  subtitle = isPending ? t('confirmingTransaction') : subtitle
+  subtitle = isStandbyTransaction ? t(STAND_BY_TRANSACTION_SUBTITLE_KEY) : subtitle
```

Comment:
I think doing it this way will prevent us from being able to do some static replace thing that I don't remember well. @jeanregisser mentioned it a few times, it's preferred if we don't pass variables to the `t` function, just strings. I think the constant is only used in the test? In that case I think we can just hard-code it there